templates/gRPC-Core.podspec.template

```diff
@@ -168,18 +174,21 @@
     s.subspec 'Interface' do |ss|
       ss.header_mappings_dir = 'include/grpc'
 
-      ss.source_files = ${ruby_multiline_list(grpc_public_headers(libs), 22)}
+      ss.source_files = ${ruby_multiline_list(grpc_public_headers, 22)}
     end
     s.subspec 'Implementation' do |ss|
       ss.header_mappings_dir = '.'
       ss.libraries = 'z'
       ss.dependency "#{s.name}/Interface", version
       ss.dependency 'BoringSSL-GRPC', '0.0.5'
+      abseil_version = '0.20190808.1'
```

Comment:
I'll keep this until they release the next LTS version of abseil supporting podspec officially. This will happen soon and we can stick with that version for a while.